packages/app-desktop/gui/utils/NoteListUtils.ts

```diff
@@ -191,9 +191,11 @@ export default class NoteListUtils {
 			const location = info.view.location;
 			if (location !== MenuItemLocation.Context && location !== MenuItemLocation.NoteListContextMenu) continue;
 
-			menu.append(
-				new MenuItem(menuUtils.commandToStatefulMenuItem(info.view.commandName, noteIds))
-			);
+			if (menuUtils.isCommandEnabled(info.view.commandName)) {
```

Comment:
Just call `cmdService.isEnabled` directly, no need to create a helper function that does nothing for this.